openpype/hosts/maya/plugins/publish/extract_pointcache.py

```diff
@@ -108,7 +108,8 @@ def process(self, instance):
         }
         instance.data["representations"].append(representation)
 
-        instance.context.data["cleanupFullPaths"].append(path)
+        if not instance.data.get("stagingDir_persistent", False):
```

Comment:
Why we do combine camel and snake case? `stagingDirIsPersistent` or `stagingDirPersist`

Reply:
This is a hiccup from the original implementation: https://github.com/ynput/OpenPype/pull/4583

Do you want it changed?

Reply:
No.